build/artifacts.props

```diff
@@ -2,7 +2,8 @@
 
   <ItemDefinitionGroup>
     <PackageArtifact>
-      <Metapackage>false</Metapackage>
+      <AppMetapackage>false</AppMetapackage>
```

Comment:
Since these cannot both be true, can you add a check to ensure this? Similar to this check:
https://github.com/aspnet/Universe/blob/cedbbd83943770eaefc79e67666cb7365952b0f3/build/repo.targets#L181-L184

Or, you could make this an enum instead of two bools `Metapackage="all"`, `Metapackage="app"`

Reply:
Checks added